### PR TITLE
Minor fixes for PHP 8+ compatibility and Roundcube 1.5.0 & add Turkish translations

### LINKED ISSUE #60: Turkish localization files
Hi,
There are Turkish language files for all sub components in attached file.
enjoy it!
Furkan
[ispconfig3_roundcube_tr_TR.tar.gz](https://github.com/w2c/ispconfig3_roundcube/files/662061/ispconfig3_roundcube_tr_TR.tar.gz)

### PR DESCRIPTION
**Pull request by GwynethLlewelyn**
Hi there!

To my surprise (since it's not officially supported), all these plugins work almost flawlessly with PHP 8 and Roundcube 1.5.0. The changes are minor and only threw a few warnings; but the warnings of today are the deprecations of tomorrow, so I added a few `empty()`s around to make sure that no `null`s pop up unexpectedly in arrays and/or class names.

I've also added the Turkish translations provided by #60.

The README was slightly updated (adding logos and such) with some additional instructions for those who have upgraded to the (yet) unsupported Ubuntu 22.04, which has, by default, a broken `php8.0-soap` extension which deliberately gets _not_ installed. The workaround is to install the always-recommended Ondrej PHP PPA.

Closes #60

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+.apdisk
+
+# Stuff from the Nova editor
+.nova
+node_modules
+package.json
+package-lock.json
+.eslintrc.yml
+.prettierrc.json
+
```
Comment (thompol) [inline .gitignore:41]: .editorconfig might be added here

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[BSD 3-Clause License](https://choosealicense.com/licenses/bsd-3-clause/)
+
+Copyright (c) 2007-2020, Till Brehm, ISPConfig UG
```
Comment (thompol) [inline LICENSE.md:3]: These plugins are not maintained by the ISPConfig developers, so no copyright for us.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,15 @@
-ispconfig3_roundcube
-====================
+![ISPConfig3 Logo](ispconfig-logo.png)
```
Comment (thompol) [inline README.md:1]: We should not add the logo as we do not maintain these plugins.

**Comment by Churro** [comment]
Hi @GwynethLlewelyn, thank you so much for your contribution!

I've now cherry-picked your commit with Turkish translations. Your findings on deprecations with PHP >= 8.0 make total sense and it's definitely worth to have them addressed. While testing them, I found further spots where variables could be uninitialized and in order not to miss any, simply went over all files. A cross check using PHP 8.1 showed no more deprecations or warnings. 

For easier maintenance we keep docs in Wiki. I've added the infos you shared on Ubuntu 22.04 LTS there:
https://github.com/w2c/ispconfig3_roundcube/wiki/System-Requirements#setup-on-ubuntu-2204-lts-with-php-80

**Comment by GwynethLlewelyn** [comment]
Thanks, guys :) It was a pleasure to help you out, within my abilities!

**Pull request closed without merging**